wire/msgdoneibdblocks.go

```diff
@@ -0,0 +1,42 @@
+package wire
+
+import (
+	"io"
+)
+
+// MsgDoneIBDBlocks implements the Message interface and represents a kaspa
+// DoneIBDBlocks message. It is used to notify the IBD syncing peer that the
+// syncer sent all the requested blocks.
+//
+// This message has no payload.
+type MsgDoneIBDBlocks struct{}
+
+// KaspaDecode decodes r using the kaspa protocol encoding into the receiver.
+// This is part of the Message interface implementation.
+func (msg *MsgDoneIBDBlocks) KaspaDecode(r io.Reader, pver uint32) error {
```

Comment:
If you are returning nil, I guess you are relying on NOD-1191 to get merge before this one is.
Therefore once it is merged, you should remove KaspaEncode/Decode and any other relevant methods